src/app/components/Image/index.tsx

```diff
@@ -124,6 +126,7 @@ const Image = ({
         ]}
         style={{
           paddingBottom: hasFixedAspectRatio ? legacyBrowserAspectRatio : 0,
+          overflow: hasCaption ? '' : 'hidden',
```

Comment:
Does applying the style this resolve the stylesheet size issue? Is there a chance the ternary statement on lines 115/116/117 added a load more classes? (Is there a way to check?) 😬

Reply:
Yeah that resolved the size issue, i think because it took it out of emotion css weirdness , emotion generates a bunch of classes - very possible that ternary is adding  a bunch! idk how or if we can check it though

Reply:
Cool yep interesting - will make a note to look at it!